kong/conf_loader/init.lua

```diff
@@ -1228,17 +1228,10 @@ local function check_and_parse(conf, opts)
     end
   end
 
-  if conf.wasm_modules then
-    for _, path in ipairs(conf.wasm_modules) do
-      local extension = pl_path.extension(path)
-
-      if not pl_path.isfile(path)
-         or not pl_path.exists(path)
-         or extension ~= ".wasm"
-      then
-        errors[#errors + 1] = fmt("wasm_module at '%s' not .wasm or " ..
-                                  "does not exists", path)
-      end
+  if conf.wasm and conf.wasm_filters_path then
```

Comment:
Can we split these into nested if blocks or better create a new "validate_wasm" to limit the length of this already long function?